include/grpcpp/security/tls_credentials_options.h

```diff
@@ -55,12 +55,12 @@ class TlsKeyMaterialsConfig {
   }
   int version() const { return version_; }
 
-  /** Setter for key materials that will be called by the user. The setter
-   * transfers ownership of the arguments to the config. **/
-  void set_pem_root_certs(grpc::string pem_root_certs);
+  /** Setter for key materials that will be called by the user. **/
```

Comment:
Please add some comments about the ownership of key materials.

Reply:
Done.